.gitignore

```diff
@@ -67,6 +67,8 @@ secrets
 # ts build info and report
 catalog-rest-service/src/main/resources/ui/webpack
```

Comment:
@darth-coder00 You can remove these path and keep it clean

Reply:
Yes